source/renderer/app/components/sidebar/SidebarCategory.tsx

```diff
@@ -5,38 +5,52 @@ import SVGInline from 'react-svg-inline';
 import { observer } from 'mobx-react';
 import classNames from 'classnames';
 import { camelCase } from 'lodash';
+import { PopOver } from 'react-polymorph/lib/components/PopOver';
+import { injectIntl } from 'react-intl';
 import type { SidebarCategoryInfo } from '../../config/sidebarConfig';
 import styles from './SidebarCategory.scss';
+import { messages } from './SidebarCategory.messages';
+import type { Intl } from '../../types/i18nTypes';
 
 type Props = {
   category: SidebarCategoryInfo;
+  intl: Intl;
   isActive: boolean;
   onClick: (...args: Array<any>) => any;
   content?: Node;
 };
 
-@observer
-class SidebarCategory extends Component<Props> {
-  render() {
-    const { category, isActive, onClick, content } = this.props;
-    const { name, icon, route } = category;
-    const className = camelCase(name);
-    const componentStyles = classNames(
-      styles.component,
-      className,
-      styles[className],
-      {
-        [styles.active]: isActive,
-      }
-    );
-    const iconClassName = classNames(styles.icon, styles[`${className}Icon`]);
-    return (
+function SidebarCategory({
+  category,
+  intl,
+  isActive,
+  onClick,
+  content,
+}: Props) {
+  const { name, icon, route, tooltipTextId } = category;
+  const className = camelCase(name);
+  const componentStyles = classNames(
+    styles.component,
+    className,
+    styles[className],
+    {
+      [styles.active]: isActive,
+    }
+  );
+  const iconClassName = classNames(styles.icon, styles[`${className}Icon`]);
+  return (
+    <PopOver
+      delay={[100, 500]}
+      offset={[0, -20]}
+      content={intl.formatMessage(messages[tooltipTextId])}
+      placement="bottom"
+    >
       <button className={componentStyles} onClick={() => onClick(route)}>
         <SVGInline svg={icon} className={iconClassName} />
         {content}
       </button>
-    );
-  }
+    </PopOver>
+  );
 }
 
-export default SidebarCategory;
+export default injectIntl(observer(SidebarCategory));
```

Comment:
You can also do it this way I think:

`const { formatMessage } = useIntl()` or
`const intl = useIntl()`

Reply:
Not in the version of `react-intl` we have. We're 3 major versions behind.
They provide upgrade guide for each major version (eg. https://formatjs.io/docs/react-intl/upgrade-guide-3x) - maybe it would be worth to give it a go?

Reply:
Never mind. :)